NICE.Identity.Management/ClientApp/src/views/UsersList/UsersList.tsx

```diff
@@ -94,12 +109,36 @@ export class UsersList extends Component<UsersListProps, UsersListState> {
 		this.setState({ isLoading: true });
 
 		const users = await fetchData(Endpoints.usersList);
+		const services = await fetchData(Endpoints.servicesList);
 
 		if (isDataError(users)) {
 			this.setState({ error: users });
 		}
 
-		this.setState({ originalUsers: users, users, isLoading: false });
+		if (isDataError(services)) {
+			this.setState({ error: services });
+		}
+
+		const servicesByEnvironment = services.reduce(
```

Comment:
What happens here if you have an error on fetching services? I think you have tests for what happens for if there is an error fetching users but not services.